scripts/linters/warranted_angular_security_bypasses.py

```diff
@@ -30,8 +30,8 @@
     # SVG is treated as unsafe by default by Angular. In order to show SVGs, we
     # have to manually sanitize the SVG and mark the value as safe. Marking a
     # value as a safe is done by bypassing the inbuilt Angular's security
-    # mechanism. The svg-sanitizer file is going to be a permanent member in
+    # mechanism. The image-sanitizer file is going to be a permanent member in
     # this list due to the aforementioned reason.
-    'core/templates/services/svg-sanitizer.service.spec.ts',
-    'core/templates/services/svg-sanitizer.service.ts')
+    'core/templates/services/image-sanitizer.service.spec.ts',
+    'core/templates/services/image-sanitizer.service.ts')
```

Comment:
The only commonly used type of image source that is treated as dangerous by Angular is SVG. All the others have reasonable checks and we need to follow those. Please change this back to `svg-sanitizer`.

Reply:
Done.